deployment/ansible/roles/raster-foundry.jenkins/handlers/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Restart Jenkins
+  service: name=jenkins state=restarted
+
+- name: Restart Nginx
```

Comment:
Fixed in [`564b758`](https://github.com/azavea/raster-foundry/pull/419/commits/564b758563a02b4b95630378f8995edeaa9c49dc)